src/S3Bucket.ts

```diff
@@ -19,6 +19,23 @@ export const getS3Client = () => {
       forcePathStyle: true,
       region: config.s3.region,
     });
+
```

Comment:
I think this is unrelated, and should probably be moved to a separate branch. Also, would it not be easier using [this param](https://aws.amazon.com/blogs/developer/the-aws-sdk-for-javascript-now-supports-amazon-s3-requester-pays-buckets/)?

Reply:
I swear I was googling something like this for ages yesterday and never found it 😭 

But yes I'll move it to a separate branch 👍